packages/git/src/common/git-model.ts

```diff
@@ -65,6 +70,7 @@ export namespace WorkingDirectoryStatus {
                 && (left.aheadBehind ? left.aheadBehind.ahead : -1) === (right.aheadBehind ? right.aheadBehind.ahead : -1)
                 && (left.aheadBehind ? left.aheadBehind.behind : -1) === (right.aheadBehind ? right.aheadBehind.behind : -1)
                 && left.changes.length === right.changes.length
+                && left.incomplete === right.incomplete
```

Comment:
I have just realized, it is not completely correct. `true` and `undefined` should be the same as you have stated in the comment: _Otherwise, (including `undefined`) is complete._

Reply:
Sorry, I don't get your point.
If the limit is reached `incomplete` is `true`. If `incomplete` is `false` or `undefined` we know that the result is complete. 
So, as far as I understood, `false` and `undefined` are the same.

Reply:
```
$ node
> console.log(false === undefined)
false
undefined
> .exit
```

Reply:
Ah, sorry. I was confused from 

> `true` and `undefined` should be the same...